src/Compilers/Core/Portable/Operations/IDeclarationPatternOperation.cs

```diff
@@ -15,6 +15,13 @@ namespace Microsoft.CodeAnalysis.Operations
     /// </remarks>
     public interface IDeclarationPatternOperation : IPatternOperation
```

Comment:
Add `ITypeSymbol MatchedType { get; }`

Reply:
`null` when `var`

---
In reply to: [243433593](https://github.com/dotnet/roslyn/pull/31967#discussion_r243433593) [](ancestors = 243433593)